src/frontends/paddle/src/op/set_value.cpp

```diff
@@ -0,0 +1,180 @@
+
+    Output<Node> starts_node, ends_node, steps_node, starts, ends, steps;
+
+    // The following process is:
+    // Given:
+    // input_data: shape(5, 6, 7, 8, 9)
+    // update_value: shape(1, 6, 3, 3)
+    // operation: input_data[:, :, 2: 7: 2, -4: -1] = update_value
+    // axes = [2, 3]
+    // starts = [2, -4]
+    // ends = [7, -1]
+    // steps = [2, 1]
+    // Our process is:
+    // 1. Get axes [2, 3], get shape of input [5, 6, 7, 8, 9], select dimension from shape by axes: [7, 8].
+    // 2. Get starts [2, -4] and ends [3, -1]. Process minus starts and ends. starts: [2, 4], ends: [7, 7].
+    // 3. Calculate starts_node, ends_node and steps_node
+    //    1. Create `starts node` filled with 0. Update `starts` to `starts_node` according to axes.
+    //    starts_node[axes[i]] = starts[i] for i in axes.size
+    //    starts_node: [0, 0, 0, 0, 0] -> [0, 0, 2, 4, 0].
+    //    2. Update `ends` to `input_shape` according to axes.
+    //    input_shape[axes[i]] = ends[i] for i in axes.size
+    //    ends_node: [5, 6, 7, 8, 9] -> [5, 6, 7, 7, 9].
+    //    3. Create `steps_node` filled with 1. Update `steps' to `steps_node` according to axes.
+    //    steps_node[axes[i]] = steps[i]  for i in axes.size
+    //    steps_node: [1, 1, 1, 1, 1] -> [1, 1, 2, 1, 1].
+    // 4. Calculate and broadcast update_value to corresponding shape: [5, 6, 5, 3, 9].
+    //    1. Calculate `end - start`: [5, 3].
+    //    2. Calculate `(end - start) / abs(step)`: [2.5, 3]
+    //    3. Calculate `ceil((end - start) / step)`: [3, 3]
+    //    2. Use `ScatterNDUpdate` to get `target_value_shape` for input_shape: [5, 6, 7, 8, 9] -> [5, 6, 3, 3, 9].
+    //    3. Broadcast from [1, 6, 3, 3] to [5, 6, 3, 3, 9]
+    // 5. Create a range_node filled with number from 0 to numel.
+    //    1. Reshape it to input_shape.
+    //    2. Use `StridedSlice` to get those indices which are about to be updated.
+    // 6. Flatten input, update_value and sliced_range.
+    // 7. Use `ScatterUpdate` update update_value into input_data.
+    // 8. Reshape input to original input_shape.
+
+    const auto axes_node = default_opset::Constant::create(element::i64, {axes.size(), 1}, axes);
+    const auto spec_dim_node = std::make_shared<default_opset::GatherND>(input_shape, axes_node);
+    const auto zero_node = default_opset::Constant::create(element::i64, Shape{}, {0});
+    const auto one_node = default_opset::Constant::create(element::i64, Shape{}, {1});
+    const auto dim_node = default_opset::Constant::create(element::i64, Shape{}, {dims});
+    const auto reshape_flatten = default_opset::Constant::create(ov::element::i64, {1}, {-1});
+    const auto slice_shape = default_opset::Constant::create(ov::element::i64, {1, 1}, {-1});
+
+    // get positive starts ends and steps
+    if (node.has_input("StartsTensorList") && node.has_input("StepsTensorList") && node.has_input("EndsTensorList")) {
```

Comment:
If have no good solution to check the negative number in starts/ends input tensor, can remove the support of starts/ends inputs currently. Will the testcase use the starts/ends input or attribute?